requirements.txt

```diff
@@ -8,5 +8,6 @@ tqdm
 datasets
 requests
 optuna
-pydantic==2.5.0
-# To be deleted
\ No newline at end of file
+pydantic==2.5.0,<2.5.3
+mistralai
```

Comment:
remove mistral and refer to comment above on pydantic for updates

Reply:
I removed it